src/ctapipe/reco/stereo_combination.py

```diff
@@ -405,3 +415,251 @@ def predict_table(self, mono_predictions: Table) -> Table:
         stereo_table[f"{self.prefix}_telescopes"] = tel_ids
         add_defaults_and_meta(stereo_table, _containers[self.property], self.prefix)
         return stereo_table
+
+
+class StereoDispCombiner(StereoCombiner):
```

Comment:
How does this algorithm differ from the implementation in EventDisplay by @GernotMaier ?

Reply:
> How does this algorithm differ from the implementation in EventDisplay by @GernotMaier ?

In general theses algorithms are very similar - ED instead calculates the minimum distance for all permutations of the 5 best (weighted) telescope events. E.g. for an Event with 8 valid telescopes:
 - Taking the 5 brightest telescope events (or any other weighting)
 - Generate all permutations of both signs for these 5 telescope events
 - Calculate distance for each permuation
 - Average permutation with minimum distance

Reply:
Could we enable using the EventDisplay definition with come configuration options? E.g. a `max_telescopes` and `telescope_priority_criterion` or similar? This would be great for performance studies to directly compare to EventDisplay

Reply:
In general yes, but this would need some severe refactoring of this PR at least for the table-wise processing. We have a Masters Student now (@AkramAki) who is working on that and would implement it (including maybe also a KMeans and DBScan based algorithm) the next months/next year. I would still suggest to merge the Combiner in the current form and do the refactoring and additional algorithms in separate PRs later on.